src/os_auth/main-server.c

```diff
@@ -526,6 +526,13 @@ int main(int argc, char **argv)
             fp = fopen(AUTHD_PASS, "r");
             buf[0] = '\0';
 
+            fseek(fp, 0, SEEK_END);
```

Comment:
I'm not sure that doing this will allow later to read correctly this file. Please retest and try to find a different solution.